ansible/roles/prometheus-server/defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+# Prometheus
+prometheus_version: 2.20.1
+prometheus_tar_gz: "prometheus-{{ prometheus_version }}.linux-amd64.tar.gz"
+prometheus_tar_gz_url: "https://github.com/prometheus/prometheus/releases/download/v{{ prometheus_version }}/{{ prometheus_tar_gz }}"
```

Comment:
prometheus_version: 2.20.1
prometheus_tar_gz: "prometheus-{{ prometheus_version }}.linux-amd64.tar.gz"
prometheus_tar_gz_url: "https://github.com/prometheus/prometheus/releases/download/v{{ prometheus_version }}/{{ prometheus_tar_gz }}"

Reply:
OK, done.
Note that I couldn't get that to work following that example - I had to also configure AWS credential parameters - aws_s3 documentation seemed to suggest this should happen automatically, but didn't for me.